pandas/tests/series/test_analytics.py

```diff
@@ -1265,12 +1273,19 @@ def test_idxmax(self):
         result = s.idxmin()
         assert result == 1.1
 
-    def test_numpy_argmax(self):
-
-        # argmax is aliased to idxmax
+    def test_numpy_argmax_deprecated(self):
```

Comment:
Same comment as <a href="https://github.com/pandas-dev/pandas/pull/16955#discussion_r128039242">above</a> for `argmin`.